src/nvim/message.h

```diff
@@ -75,6 +77,13 @@
 /// Like #MSG_PUTS_ATTR, but if middle part of long messages will be replaced
 #define MSG_PUTS_LONG_ATTR(s, a)    msg_puts_long_attr((char_u *)(s), (a))
 
+typedef struct {
```

Comment:
hmm probably we want to change virt text to use the same type, but I can do it as a follow-up.